daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1595,12 +1618,28 @@ func (devices *DeviceSet) Status() *Status {
 
 		status.Data.Used = dataUsed * blockSizeInSectors * 512
 		status.Data.Total = dataTotal * blockSizeInSectors * 512
+		status.Data.Available = status.Data.Total - status.Data.Used
 
 		// metadata blocks are always 4k
 		status.Metadata.Used = metadataUsed * 4096
 		status.Metadata.Total = metadataTotal * 4096
+		status.Metadata.Available = status.Metadata.Total - status.Metadata.Used
 
 		status.SectorSize = blockSizeInSectors * 512
+
+		if check, _ := devices.isRealFile(devices.dataLoopFile); check {
```

Comment:
This check of whether it is a real loop file is not needed now. If `devices.dataLoopFile != ""` then there is a loop file.